op-batcher/batcher/config.go

```diff
@@ -185,6 +183,14 @@ func (c *CLIConfig) Check() error {
 	if err := c.RPC.Check(); err != nil {
 		return err
 	}
+
+	if c.Espresso.L1URL == "" {
+		c.Espresso.L1URL = c.L1EthRpc
```

Comment:
Should we add a log here in case the `L1URL` is supposed to be different than `L1EthRpc` but we forgot to set it?

Reply:
Sure, makes sense